src/main/resources/static_config/static_roles.yml

```diff
@@ -20,6 +20,8 @@ all_access:
         - "*"
       allowed_actions:
         - "kibana_all_write"
+  extension_permissions:
```

Comment:
Are we going to be adding a new field to all roles designated as extension permissions?